pkgs/by-name/x2/x2t/package.nix

```diff
@@ -0,0 +1,357 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  libxml2,
+  boost,
+  gumbo,
+  icu,
+  freetype,
+  qt5,
+  harfbuzz,
+  # needs to be static and built with MD2 support!
+  openssl,
+  # should find out if newer versions work...
+  v8,
```

Comment:
`v8` has been (rightfully) dropped from nixpkgs since, so we should really see if we can support running with a newer v8. Newer v8's are available via `nodejs.libv8`, but I've already verified it's not a drop-in replacement: some work will be needed to be able to use it. Hopefully not too much :crossed_fingers: